lib/AppInfo/Application.php

```diff
@@ -178,8 +179,8 @@ private function registerFilesActivity(IRegistrationContext $context) {
 		Util::connectHook('OC_Filesystem', 'rename', FilesHooksStatic::class, 'fileMove');
 		Util::connectHook('OC_Filesystem', 'post_rename', FilesHooksStatic::class, 'fileMovePost');
 		Util::connectHook('\OCA\Files_Trashbin\Trashbin', 'post_restore', FilesHooksStatic::class, 'fileRestore');
-		Util::connectHook('OCP\Share', 'post_shared', FilesHooksStatic::class, 'share');
```

Comment:
@artonge yes the call was here. But it is removed and replaced by the event listener below